Sources/EdgeNetworkHandlers/EdgeNetworkService.swift

```diff
@@ -27,23 +27,21 @@ enum HttpResponseCodes: Int {
     case ok = 200
     case noContent = 204
     case multiStatus = 207
-    case clientTimeout = 408
     case tooManyRequests = 429
     case badGateway = 502
-    case serviceUnavailable = 503
-    case gatewayTimeout = 504
+    case insufficientStorage = 507
 }
 
 /// Network service for requests to the Adobe Experience Edge
 class EdgeNetworkService {
     private let SELF_TAG: String = "EdgeNetworkService"
     private let DEFAULT_GENERIC_ERROR_MESSAGE = "Request to Experience Edge failed with an unknown exception"
     private let DEFAULT_GENERIC_ERROR_TITLE = "Unexpected Error"
-    private let recoverableNetworkErrorCodes: [Int] = [HttpResponseCodes.clientTimeout.rawValue,
-                                                       HttpResponseCodes.tooManyRequests.rawValue,
-                                                       HttpResponseCodes.badGateway.rawValue,
-                                                       HttpResponseCodes.serviceUnavailable.rawValue,
-                                                       HttpResponseCodes.gatewayTimeout.rawValue]
+    private let recoverableNetworkErrorCodes: [Int] = NetworkServiceConstants.RECOVERABLE_ERROR_CODES +
```

Comment:
Since this is private, can you convert it to a set to simplify the logic?
```
Set(NetworkServiceConstants.RECOVERABLE_ERROR_CODES +
                [HttpResponseCodes.tooManyRequests.rawValue,
                HttpResponseCodes.badGateway.rawValue,
                HttpResponseCodes.insufficientStorage.rawValue]
```